### PR TITLE
LPS-189320 Remove Feature Flag LPS-153117

### PR DESCRIPTION
**Pull request by sergiojimcos**
https://liferay.atlassian.net/browse/LPS-189320

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by sergiojimcos** [comment]
ci:test:relevant

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/extension/v1_0/ObjectRelationshipExtensionProvider.java [hunk 1]
```diff
@@ -174,15 +173,6 @@ public Map<String, PropertyDefinition> getExtendedPropertyDefinitions(
 				continue;
 			}
 
-			PropertyValidator propertyValidator = null;
-
-			if (FeatureFlagManagerUtil.isEnabled("LPS-153117")) {
-				propertyValidator = new DefaultPropertyValidator();
-			}
-			else {
-				propertyValidator = new UnsupportedOperationPropertyValidator();
```
Comment (ccorreagg) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/extension/v1_0/ObjectRelationshipExtensionProvider.java:183]: I think the private class `UnsupportedOperationPropertyValidator` could be removed as I think we are not using it anymore

**Review by ccorreagg — COMMENTED** [review]
just a comment!

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/extension/v1_0/ObjectRelationshipExtensionProvider.java (hunk 1 above)
Comment (sergiojimcos) [reply modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/extension/v1_0/ObjectRelationshipExtensionProvider.java:183]: :facepalm: that's true, thanks @ccorreagg!!

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Comment by sergiojimcos** [comment]
ci:forward

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
Test failures are not related. OK to manually forward ✅

**Comment by sergiojimcos** [comment]
ci:forward:force

**Pull request closed without merging**